FloripaTrans/FTFindRoutesGoogleMapsViewController.m

```diff
@@ -0,0 +1,130 @@
+//
+//  FTFindRoutesGoogleMapsViewController.m
+//  FloripaTrans
+//
+//  Created by Vítor Makoto Matayoshi de Moraes on 3/7/15.
+//  Copyright (c) 2015 Vitor Makoto. All rights reserved.
+//
+
+#import "FTFindRoutesGoogleMapsViewController.h"
+#import <GoogleMaps/GoogleMaps.h>
+#import <CoreLocation/CoreLocation.h>
+#import "UIView+FTViewLayout.h"
+
+@interface  FTFindRoutesGoogleMapsViewController() <GMSMapViewDelegate>
+@property (weak, nonatomic) IBOutlet GMSMapView *mapView;
+@property (strong, nonatomic) GMSMarker *routeMarker;
+
+@property (weak, nonatomic) IBOutlet UIButton *okButton;
+
+@end
+@implementation FTFindRoutesGoogleMapsViewController
+
+- (void)viewDidLoad {
+    
+    
+    self.navigationItem.title = @"Map";
+    
+    // Set up the map view
+    self.mapView.myLocationEnabled = NO;
+    self.mapView.mapType = kGMSTypeNormal;
+    self.mapView.settings.compassButton = YES;
+    self.mapView.settings.myLocationButton = NO;
+    
+    self.mapView.delegate = self;
+    
+    // Round the corner of the search button
+    [self.okButton roundViewCornerWithRadius:10];
+    
+    // Set up initial location for the map view
+    GMSCameraPosition *camera = [GMSCameraPosition cameraWithLatitude:-27.593500
+                                                            longitude:-48.558540
+                                                                 zoom:14];
+    self.mapView.camera = camera;
+    
+    // Instantiate the route marker
+    self.routeMarker = [[GMSMarker alloc] init];
+    
+    // Change the back button so it has the title Back
+    UIBarButtonItem *item = [[UIBarButtonItem alloc] initWithTitle:@"Back" style:UIBarButtonItemStylePlain target:self action:@selector(goBack)];
+    self.navigationItem.hidesBackButton = YES;
+    self.navigationItem.leftBarButtonItem = item;
+    
+}
+
+- (void)goBack {
+    [self.navigationController popViewControllerAnimated:YES];
+}
+
+- (IBAction)okTouchUp:(id)sender {
+    if(!self.routeMarker.map) {
+        [self alertMessage:@"Choose a location" withTilte:@"Location Selection"];
```

Comment:
Yeah, I'm going to try to fix them all. I will open an Issue for this. Thanks for the advice!